index.html

```diff
@@ -129,6 +129,13 @@ <h4>Student Support</h4>
                     </li>
                   </ul>
                 </li>
+                <li><a href="https://github.com/joeyish" target="_blank" rel="noopener noreferrer">Joseph Bishop</a>
```

Comment:
```suggestion
                <li><a href="https://github.com/joeybish" target="_blank" rel="noopener noreferrer">Joseph Bishop</a>
```
Missing a "b" here. Also, I'd keep the first name matching the bottom one as well, just to keep things consistent.